_posts/2017-04-13-bell-hooks-feminism-is-for-everybody.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: posts
+title: "Feminism is for Everybody by bell hooks"
+date:   2017-04-13
+permalink: "/feminism-is-for-everybody-bell-hooks/"
+categories: feminist-book-club
+tags: feminism anti-racism anti-capitalism study-guide
```

Comment:
I think categories should be "study-guides feminist-book-club". Or maybe in reverse order?